source/docs/software/advanced-controls/controllers/feedforward.rst

```diff
@@ -11,6 +11,8 @@ There are two types of feedforwards: model-based feedforward and feedforward for
 
 .. note:: The WPILib feedforward classes closely match the available mechanism characterization tools available in the :ref:`frc-characterization toolsuite <docs/software/pathplanning/robot-characterization/introduction:Introduction to Robot Characterization>` - the characterization toolsuite can be used to quickly and effectively determine the correct gains for each type of feedforward.  The toolsuite will indicate the appropriate units for each of the gains.
 
+.. note:: If you are unable to empirically characterize your mechanism (due to space and/or time constraints), reasonable estimates of ``kG``, ``kV``, and ``kA`` can be obtained by fairly simple computation, and are also available from `ReCalc<https://www.reca.lc>`__.
```

Comment:
I don't like two consecutive notes. It seems like this could be added to the previous note, or better yet expanded into its own section